content/pages/disability-benefits/after-you-apply/VA-claim-exam.md

```diff
@@ -77,10 +77,10 @@ If you can’t make it to your appointment, let us know right away. You can most
 
 **To reschedule your appointment:**
 
-- Call 1-800-827-1000, **or**
-- Go to your nearest regional benefit office. <br>
-  [Find your local regional benefit office](/facilities/).	
-  
+* Call 1-800-827-1000, **or**
+* Go to your nearest regional benefit office. <br>
+  [Find your local regional benefit office](/facilities/).
+
```

Comment:
Just wondering, what's the difference between using `*` versus a `-` for creating lists?

Reply:
there's no major difference, it's my prettier formatting it into a default practice syntax :)